components/Typeahead/_typeahead.scss

```diff
@@ -6,72 +6,108 @@
 @include exports('typeahead') {
   .bx--typeahead {
     position: relative;
-    display: flex;
   }
 
-  .bx--typeahead-clear {
-    @include button-reset(false);
-    align-self: center;
-    position: absolute;
-    right: 45px;
-    cursor: pointer;
+  .bx--typeahead__input {
+    padding-right: rem(70px);
+  }
+
+  // Apparently, in IE10+ there is a "clear" icon added to inputs that we
+  // need manually hide since we use our own icon to clear.
+  .bx--typeahead__input::-ms-clear {
+    display: none;
```

Comment:
Should this also have a `visibility: hidden` for a11y?

Reply:
Sure! Will add in.

Reply:
It seems like they both accomplish the same thing in regards to screen readers, but might as well add it just to be safe. 

https://webaim.org/techniques/css/invisiblecontent/